pages.pt_BR/common/fd.md

```diff
@@ -0,0 +1,29 @@
+# fd
+
+> Uma alternativa para `find`.
+> Visa ser mais rápido e fácil de usar do que `find`.
+> Mais informações: <https://github.com/sharkdp/fd>.
+
+- Encontra recursivamente arquivos que correspondam ao padrão fornecido no diretório atual:
+
+`fd {{padrão}}`
+
+- Encontra arquivos que começam com `foo`:
+
+`fd '^foo'`
+
+- Encontra arquivos com uma extensão específica:
+
+`fd --extension txt`
+
+- Encontra arquivos em um diretório específico:
+
+`fd '{{padrão}}' {{caminho/para/diretório}}`
+
+- Inclui arquivos ignorados e ocultos na pesquisa:
```

Comment:
```suggestion
- Inclue arquivos ignorados e ocultos na pesquisa:
```

Reply:
Hi, thank you for the suggestions. I believe `Inclue` is not a word in pt_BR, `Inclui` is  the imperative affirmative form of the `Incluir` verb.

Reply:
For Romance languages, the present indicative fits better. `inclue` is conjugated here: https://conjugator.reverso.net/conjugation-portuguese-verb-incluir.html

Reply:
These conjugations are in pt_PT. Here are the conjugations for the verb `incluir` in pt_BR: https://www.conjugacao.com.br/verbo-incluir/